packages/flutter/lib/src/foundation/stack_frame.dart

```diff
@@ -0,0 +1,209 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show hashValues;
+
+import 'package:meta/meta.dart';
+
+/// A object representation of a frame from a stack trace.
+///
+/// {@tool sample}
+///
+/// For example, a caller that wishes to traverse the stack could use
+///
+/// ```dart
+/// final List<StackFrame> currentFrames = StackFrame.fromStackTrace(StackTrace.current);
+/// ```
+///
+/// To create a traversable parsed stack.
```

Comment:
I believe all text that's outside the dart code block will render above the code in dart docs. So this may look awkward there. I would rephrase this to have the prose text all above the code.

Reply:
done.